app/src/main/java/com/github/factotum_sdp/factotum/ui/directory/ContactCreationFragment.kt

```diff
@@ -247,7 +259,7 @@ class ContactCreationFragment : Fragment() {
         return viewModel.contacts.value?.find { it.username == username } == null
     }
 
-    private fun isUsernameExisitingContact(username: String): Boolean {
+    private fun isUsernameExistingContact(username: String): Boolean {
         return viewModel.contacts.value?.find { it.username == username && it.role == Role.CLIENT.name } != null
```

Comment:
.any() would be more appropriate here. But just for you to know for the next implementations, don't necessary need to modify that here, the null check makes the job.